packages/react-native/android/src/main/java/com/bugsnag/reactnative/BugsnagReactNative.java

```diff
@@ -1,103 +1,193 @@
 package com.bugsnag.reactnative;
 
+import com.bugsnag.android.Breadcrumb;
 import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.BugsnagException;
+import com.bugsnag.android.Callback;
 import com.bugsnag.android.Client;
 import com.bugsnag.android.Configuration;
+import com.bugsnag.android.InternalHooks;
+import com.bugsnag.android.Report;
 
 import android.content.Context;
+import android.support.annotation.NonNull;
+import android.util.Log;
 
-import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReactApplicationContext;
-import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
 import com.facebook.react.bridge.ReadableMap;
 
-import java.util.logging.Logger;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
 
 public class BugsnagReactNative extends ReactContextBaseJavaModule {
 
-    private ReactContext reactContext;
-    private String libraryVersion;
-    private String bugsnagAndroidVersion;
-    static final Logger logger = Logger.getLogger("bugsnag-react-native");
+    // TODO populate this (read directly from JS)
```

Comment:
my understanding is the `versions` field is read from JS when it requires information about the bugsnag-android version. the TODO is to set this value in the first place.

Reply:
ahhh got you! I was worried you wanted the native layer to read it _from_ JS! As long as this value is statically defined in on this class it will all be good.